Src/WindowsServer/WindowsServer.Shared/DomainNameRoleInstanceTelemetryInitializer.cs

```diff
@@ -1,53 +0,0 @@
-﻿namespace Microsoft.ApplicationInsights.WindowsServer
-{
-    using System;
-    using System.Globalization;
-    using System.Net;
-    using System.Net.NetworkInformation;
-    using System.Threading;
-    using Microsoft.ApplicationInsights.Channel;
-    using Microsoft.ApplicationInsights.Extensibility;
-    using Microsoft.ApplicationInsights.WindowsServer.Implementation;
-    
-    /// <summary>
-    /// A telemetry context initializer that populates device context role instance name.
-    /// </summary>
-    public class DomainNameRoleInstanceTelemetryInitializer : ITelemetryInitializer
```

Comment:
please keep this class and mark it as obsolete with the comment. Make it's implementation empty